### ISSUE TITLE
Add missing reformation day holidays

### ISSUE THREAD

**Issue by rudolfochrist** [rudolfochrist/german-holidays#3]
Some states gained reformating day as official holiday. Add reformation day to:

- [ ] Bremen (HB)
- [ ] Hamburg (HH)
- [ ] Schleswig Holstein (SH)

**Comment by rudolfochrist** [comment]
There a more holdiays missing: International Women's (e.g.). Got through them and add them!

**Comment by lino** [comment]
Hey, will you release this change eventually?

**Comment by rudolfochrist** [comment]
I don't think so. I stopped using this package myself 7 years ago. It has the lowest priority for me right now. But I merge PRs, if you want to give it a go!

**Comment by lino** [comment]
Thanks, will prepare a PR!
